Diffing_Engine/Query/DifferentProperties.cs

```diff
@@ -40,131 +40,36 @@ public static partial class Query
         [Description("Checks two BHoMObjects property by property and returns the differences.")]
         [Input("obj1", "First object to compare for differences.")]
         [Input("obj2", "Second object to compare for differences.")]
-        [Input("diffingConfig", "Config to be used for the comparison. Can set numeric tolerance, wheter to check the guid, if custom data should be ignored and if any additional properties should be ignored.")]
-        [Output("Dictionary whose key is the name of the property, and value is a tuple with the different values found in obj1 and obj2.\n" +
-            "This dictionary can be 'exploded' in the UI by using `ListDifferentProperties` method.")]
-        public static Dictionary<string, Tuple<object, object>> DifferentProperties(this object obj1, object obj2, DiffingConfig diffingConfig = null)
+        [Input("comparisonConfig", "Additional configurations to be used for the comparison.")]
+        [Output("Dictionary whose Key is the Full Name of the property that is different among the two objects, " +
+            "and whose Value is a Tuple: Item1 is the value of the property in obj1, Item2 the value in obj2." +
+            "\nThis dictionary can be 'exploded' in the UI by using the `ListDifferentProperties()` method." +
+            "\nIf no difference was found, returns null.")]
+        [PreviousVersion("5.0", "BH.Engine.Diffing.DifferentProperties(System.Object, System.Object, BH.oM.Diffing.DiffingConfig)")]
```

Comment:
```suggestion
        [PreviousVersion("5.0", "BH.Engine.Diffing.Query.DifferentProperties(System.Object, System.Object, BH.oM.Diffing.DiffingConfig)")]
```

Think this will fix the versioning issue @alelom